ppcs/ppc0013-overload-join-and-substr.md

```diff
@@ -98,11 +98,14 @@ We do not yet foresee security issues. Guidance is welcome.
 For `join`:
 
 ```perl
-# when @list contains elements with concat overloading,
-# we expect this code:
+use List::Util 'reduce';
+
+# When @list contains elements with concat overloading, we expect this code:
+
 my $ret = join $sep, @list;
 
-# to behave like this code:
+# To behave like this code:
+
```

Comment:
As it's only relevant to this line, surely put the the `use List::Util 'reduce';` down here?

Reply:
I see, yes, force-pushed.